resources/views/auth/login.blade.php

```diff
@@ -20,11 +20,11 @@ class="col-md-4 col-form-label text-md-right">{{ __('E-Mail Address') }}</label>
                                            class="form-control @error('email') is-invalid @enderror" name="email"
                                            value="{{ old('email') }}" required autocomplete="email" autofocus>
 
-                                    @error('email')
-                                    <span class="invalid-feedback" role="alert">
-                                        <strong>{{ $message }}</strong>
-                                    </span>
-                                    @enderror
+{{--                                    @error('email')--}}
+{{--                                    <span class="invalid-feedback" role="alert">--}}
+{{--                                        <strong>{{ $message }}</strong>--}}
```

Comment:
bản 5.5 ko dùng đc error kiểu này nữa r. Bản 5.5 e test toàn kiểu [ 0 => message , 1=> message , ...]. E chưa tìm đc cách cho hiển thị message error theo inline input

Reply:
Nếu mà login sai thì nó có báo đỏ ở input kiểu is-invalid . Chả cần message